### DISCUSSION TITLE
CasaOS is not disovered in Windowss 11 or Windows 10 machines

### DISCUSSION THREAD

**Discussion by Ghazzawi76** [IceWhaleTech/CasaOS#1785] [General]
HI ,

this is not a serious issue but would be great if you can point me on how to have windows 11 and windows machines in the network ( workgroup name = WORKGROUP ) discover CasaOS 

i can access CasaOS shares from the windows machines by IP (\\192.168.x.x\sharename ) but would like to see it appear in the networks of the workgroup

**Comment by Kigurame** [comment]
NetBios discovery is not supported anymore under windows. 10/11 to solve this issue you have to install [wsdd](https://github.com/christgau/wsdd) on your distribution this will enable Web Service Discovery to expose your shares.
